src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining.Api/Startup.cs

```diff
@@ -56,6 +55,7 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMediatR(c => c.RegisterServicesFromAssembly(typeof(GetEmployerRequestQuery).Assembly));
             services.AddMediatR(c => c.RegisterServicesFromAssembly(typeof(GetRoatpV2ProviderQuery).Assembly));
+            services.AddMediatR(c => c.RegisterServicesFromAssembly(typeof(GetAggregatedEmployerRequestsQuery).Assembly));
```

Comment:
should only need one of these lines here.

Reply:
Removed